src/components/studies/StudyBuilder.tsx

```diff
@@ -83,6 +83,10 @@ const useStyles = makeStyles((theme: ThemeType) => ({
     alignItems: 'center',
     backgroundColor: '#FAFAFA',
     height: '90vh',
```

Comment:
why do you need to specify the height?

Reply:
Without the height the screen looks like this: 
![Screen Shot 2021-04-20 at 2 28 47 PM](https://user-images.githubusercontent.com/31671708/115466327-c84c6f80-a1e4-11eb-8439-e6ecbc886fa1.png)